Embedded/NuttX/precompiled/micro-ROS/olimex_stm32-e407/Dockerfile

```diff
@@ -34,12 +32,14 @@ RUN apt install -y  remake \
 
 
 # Clone nuttx repo
-RUN git clone https://bitbucket.com/nuttx/nuttx.git ~/nuttx
+RUN git clone https://github.com/microROS/NuttX.git ~/nuttx 
+
 
 
 # Clone nuttx apps repo.
-RUN git clone https://github.com/microROS/apps.git ~/apps 
-RUN git -C ~/apps checkout feature/Ubuntu18Update
+RUN git clone https://github.com/microROS/apps.git ~/apps
```

Comment:
We may want to merge this https://github.com/microROS/apps/pull/4 into develop and then master first. Right?

Reply:
config folder cannot live under `apps`, that breaks NuttX structure

Reply:
Sorry, I didn't explain it.
I meant the content.
```
# Use Nuttx Math lib
CONFIG_LIBM=y
```
@qeyup tried with Ubuntu 18, and this was required. Is it not a requirement any more?

Reply:
this is necesary because in ubuntu 18 it tries to compile using the system library and fails due the version is incompatible.

Reply:
I'm not saying that the library is not required. The point is that the pull-request you've mentioned, pastes content to `apps/configs/olimex-stm32-e407/uros/defconfig`, and this is not correct. All the `defconfig` files must be in `nuttx` repo

Reply:
Yes, I do agree on the `defconfig` to be on `nuttx` repo. Just wondering if the `nuttx/configs/olimex-stm32-e407/uros/defconfig` had the libm activated. I have just checked and seems that is OK.